great_expectations/render/renderer/content_block/bullet_list_content_block.py

```diff
@@ -1,13 +1,55 @@
+import copy
+
 from .content_block import ContentBlock
 
 
+def substitute_none_for_missing(kwargs, kwarg_list):
+    """Utility function to plug Nones in when optional parameters are not specified in expectation kwargs.
+
+    Example:
+        Input:
+            kwargs={"a":1, "b":2},
+            kwarg_list=["c", "d"]
+
+        Output: {"a":1, "b":2, "c": None, "d": None}
+
+    This is helpful for standardizing the input objects for rendering functions.
+    The alternative is lots of awkward `if "some_param" not in kwargs or kwargs["some_param"] == None:` clauses in renderers.
+    """
+
+    new_kwargs = copy.deepcopy(kwargs)
+    for kwarg in kwarg_list:
+        if not kwarg in new_kwargs:
+            new_kwargs[kwarg] = None
+    return new_kwargs
+
+
 class BulletListContentBlock(ContentBlock):
     _content_block_type = "bullet_list"
 
+    # Note: I don't think we should include `column_name`, since it'll already be present in the expectation object.
     @classmethod
     def expect_column_to_exist(cls, expectation, column_name=""):
-        return [column_name + " is a required field."]
+        params = substitute_none_for_missing(
+            expectation["kwargs"],
+            ["column", "column_index"],
+        )
+
+        if params["column_index"] == None:
+            return [{
+                "template": "$column is a required field.",
+                "params": params,
+            }]
+
+        else:
+            #!!! FIXME: this works for 4th, 5th, 6th, etc, but is dumb about 1th, 2th, and 3th.
+            params["column_indexth"] = str(params["column_index"])+"th"
+            return [{
+                "template": "$column must be the $column_indexth field.",
+                "params": params,
+            }]
 
+    # Abe 2019/06/22: I haven't touched this method. It still uses old conventions that I think should be deprecated.
```

Comment:
Reading linearly so this may be about to become obvious...but which conventions?

Reply:
Just the other stuff in comments.

I propose that we 
1. resolve other issues
2. apply those patterns to the "untouched" methods. This will probably get these methods fully ship shape.
3. review those methods to make sure no other issues slip through.